scenario/SupplyScenario.ts

```diff
@@ -136,6 +137,42 @@ scenario(
   }
 );
 
+scenario(
+  'Comet#supply > base asset with token fees',
+  {
+    tokenBalances: {
+      albert: { $base: 1000 }, // in units of asset, not wei
+    },
+    filter: async (ctx) => matchesDeployment(ctx, [{ deployment: 'usdt' }])
+  },
+  async ({ comet, actors }, context, world) => {
+    // Set fees for USDT for testing
+    const USDTAdmin = await world.deploymentManager.getSigner();
+    const USDT = await world.deploymentManager.existing('USDT', await comet.baseToken(), 'goerli');
```

Comment:
You can use `world.base.network` to programmatically fetch the network instead of hardcoding it.

Reply:
Oh i see. Thank you so much! 😄

Reply:
Replaced all with world.base.network